src/plugins/integrations/gradle.ts

```diff
@@ -0,0 +1,38 @@
+import { readFileSync } from 'fs';
+import { SiloDiscoveryConfig } from '../types';
+
+const regex = [/:(.\S*):/, /name: ?'(.*?)'/];
```

Comment:
what are these regexes matching on? would be great to have a comment or a more descriptive variable name